rerun_py/rerun_sdk/rerun/_image_chroma_downsampled.py

```diff
@@ -18,31 +17,6 @@ class ImageFormat:
 
     name: str
 
-    BMP: ImageFormat
-    """
-    BMP file format.
-    """
-
-    GIF: ImageFormat
-    """
-    JPEG/JPG file format.
-    """
-
-    JPEG: ImageFormat
-    """
-    JPEG/JPG file format.
-    """
-
-    PNG: ImageFormat
-    """
-    PNG file format.
-    """
-
-    TIFF: ImageFormat
-    """
-    TIFF file format.
-    """
-
```

Comment:
from a python user's pov this will now look like we support fewer formats. We need to write something in the changelog/migration guide to clarify what's happening 🤔 
Also we should probably enable gif, bmp and tiff support as well